app/src/components/button/LButton.vue

```diff
@@ -24,7 +24,7 @@ const variants = {
     primary:
         "bg-yellow-500 ring-1 shadow-sm text-yellow-950 ring-yellow-600/60 hover:bg-yellow-400/90 active:bg-yellow-500/80 disabled:bg-yellow-200 disabled:text-yellow-500 disabled:ring-yellow-200",
     secondary:
-        "bg-white ring-1 shadow-sm text-zinc-900 ring-zinc-300 hover:bg-zinc-100 active:bg-zinc-200/70 disabled:bg-zinc-100 disabled:text-zinc-500",
+        "bg-white dark:bg-slate-800 dark:hover:bg-zinc-800/20 ring-1 dark:ring-zinc-400 shadow-sm dark:text-zinc-200 text-zinc-900 ring-zinc-300 hover:bg-zinc-100 active:bg-zinc-200/70 disabled:bg-zinc-100 disabled:text-zinc-500",
```

Comment:
I think here you should use `slate` for dark but a bit much lighter. Using `zinc` goes a bit against our design rules.

Reply:
Changed